package.json

```diff
@@ -1,30 +1,50 @@
 {
-  "name": "probot",
-  "version": "0.0.0-development",
-  "description": "a trainable robot that responds to activity on GitHub",
+  "name": "probot-ts",
+  "version": "6.0.0-typescript",
+  "description": "a trainable robot that responds to activity on GitHub. in typescript!",
   "repository": "https://github.com/probot/probot",
   "main": "lib/index.js",
   "bin": {
     "probot": "./bin/probot.js"
   },
   "scripts": {
+    "build": "rimraf lib && tsc -p tsconfig.json",
     "start": "node ./bin/probot run",
-    "test": "jest --coverage && standard && npm run doc-lint",
+    "test": "jest --coverage --forceExit && standard && npm run doc-lint",
     "doc-lint": "standard docs/**/*.md",
     "doc": "jsdoc -c .jsdoc.json",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
-    "postsemantic-release": "script/publish-docs"
+    "postsemantic-release": "script/publish-docs",
+    "prepublish": "npm run build"
   },
   "jest": {
     "setupFiles": [
       "<rootDir>/test/setup.js"
+    ],
+    "coveragePathIgnorePatterns": [
+      "<rootDir>/lib/"
+    ],
+    "moduleFileExtensions": [
+      "ts",
+      "js",
+      "json",
+      "node"
+    ],
+    "transform": {
+      ".+\\.tsx?$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+    },
+    "testMatch": [
+      "<rootDir>/test/*.(ts|js)"
+    ],
+    "testPathIgnorePatterns": [
+      "<rootDir>/test/setup.js"
     ]
   },
   "author": "Brandon Keepers",
   "license": "ISC",
   "dependencies": {
     "@octokit/rest": "15.1.7",
-    "bottleneck": "^2.0.0",
+    "bottleneck": "^1.16.0",
```

Comment:
Yep. Rolled back because 2.0 is not supporting ES5 syntax, which breaks compatibility with current FaaS providers. We can move it back once that's not an issue.